### PR TITLE
Issue #6916: migrate tests to junit5 for blocks package

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6916

Tests of the `blocks` package are migrated to Junit5.

### CODE REVIEW

**Comment by rnveach** [comment]
@pbludov file conflict

**Comment by romani** [comment]
rebased ...

File: config/import-control-test.xml [hunk 1]
```diff
@@ -20,7 +20,8 @@
 
   <!-- Till https://github.com/checkstyle/checkstyle/issues/6916 -->
   <subpackage name="checks">
-    <subpackage name="(blocks|coding|design|header|imports|indentation|javadoc)" regex="true">
+    <allow pkg="org.junit" local-only="true"/>
+    <subpackage name="(coding|design|header|imports|indentation|javadoc)" regex="true">
```
Comment (rnveach) [inline config/import-control-test.xml]: Shouldn't this have only changed blocks package? Checks was another PR.

Comment (pbludov) [reply config/import-control-test.xml]: rebased and resolved

**Comment by rnveach** [comment]
Can merge when CI passes.

**Pull request merged**